tests/unittests/expressions/parsing/test_expression_parsing.py

```diff
@@ -57,6 +57,18 @@
             "x[t-1, t+4]",
             var("x").shift([-literal(1), literal(4)]),
         ),
+        (
+            {"x"},
+            {},
+            "x[t-1+1]",
+            var("x").shift(-literal(1) + literal(1)),
```

Comment:
```suggestion
            var("x"),
```
Are simplifications expected ?

Reply:
That is an ongoing debate, should we simplify expressions online to avoid manipulating large ASTs ?

Reply:
But as it stands, the code does not do the simplification